src/app/globals.css

```diff
@@ -30,4 +30,8 @@ body {
   .text-balance {
     text-wrap: balance;
   }
+
+  .bg-tan-silver {
```

Comment:
@SousthabChakma  We might remove this file `globals.css`.  Please use tailwind css to style your component.